vendor/github.com/containers/libhvee/pkg/ignition/config.go

```diff
@@ -0,0 +1,80 @@
+package ignition
+
+import "errors"
+
+const (
+	// kvpValueMaxLen is the maximum real-world length of bytes that can
+	// be stored in the value of the wmi key-value pair data exchange
+	kvpValueMaxLen = int(990)
```

Comment:
Why 990?

Reply:
because more seems to fail?

Reply:
If the failure (error code, etc.) doesn't give us anything to go on, let's at least describe the failure mode in the comment and say that we don't know why it happens.

Also, is this bytes, or UTF-16 characters?